doc/source/notebooks/advanced/varying_noise.pct.py

```diff
@@ -204,10 +207,8 @@ def objective_closure():
 set_trainable(model.kernel, False)
 set_trainable(model.likelihood, False)
```

Comment:
Is it necessary to set these to `trainable=False`? They are not being optimised as far as I can tell?